README.md

```diff
@@ -126,4 +82,46 @@ each of the following variables:
 | `flask seed all`      | Just a helpful syntax to run queries against the db to seed data. See the **app/seeds** folder for reference and more details |
 | `heroku login -i`      | Authenticate your heroku-cli using the command line. Drop the -i to authenticate via the browser |
 | `heroku authorizations:create` | Once authenticated, use this to generate an Oauth token |
-| `heroku run -a <app name>` | Run a command from within the deployed container on Heroku |
\ No newline at end of file
+| `heroku run -a <app name>` | Run a command from within the deployed container on Heroku |
+
+
+
+
+### Dev Containers (OPTIONAL for M1 Users)
```

Comment:
I don't think this section is necessary since students don't use Postgres in development anymore.